easybuild/easyconfigs/g/GPAW/GPAW-1.2.0-customize.patch

```diff
@@ -0,0 +1,36 @@
+--- gpaw-1.2.0-orig/customize.py	2017-05-24 11:57:38.765096571 +0200
```

Comment:
@mikstr Please include a short description for this patch at the top + also mention author (+ organisation) an/or include a reference to an upstream patch if relevant; anything above the first `---` line is considered comments